### PR TITLE
Add option to accept self signed certificates

### LINKED ISSUE #5: HTTP Client- SSL validation
HTTP Client should has an options sslCertValidation.
In my case rancher endpoint got an private CA.
image

### PR DESCRIPTION
**Pull request by piechpatrick**
TlsSkipVerification

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,23 +23,25 @@
...
-| rancher_url          |    ✔     |               | Base URL of the Rancher                                                                              |
-| rancher_workloads    |    ✔     |               | Comma separated list of workloads to be updated (e.g. deployment1,deployment2)                       |
+| Variable              | Required | Default value | Description                                                                                          |
+|-----------------------|:--------:|---------------|------------------------------------------------------------------------------------------------------|
+| debug                 |          | 'false'       | Debug flag (useful when something fails)                                                             |
+| disable_output        |          | 'false'       | Disables outputting to stdout (useful if the logs are public, but you don't want to expose anything) |
+| tls_skip_verification |          | 'false'       | Skipping host TLS verification                                                                       |
```
Comment (th0th) [inline README.md]:
```suggestion
| tls_skip_verification |          | 'false'       | Skips TLS verification for the outgoing HTTP request to Rancher                                      |
```

File: action.yml [hunk 2]
```diff
@@ -8,6 +8,8 @@ inputs:
     description: "Debug flag"
   disable_output:
     description: "Disables outputting to stdout (useful if the logs are public, but you don't want to expose anything)"
+  tls_skip_verification:
+    description: "Skipping host TLS verification"
```
Comment (th0th) [inline action.yml]:
```suggestion
    description: "Skips TLS verification for the outgoing HTTP request to Rancher"
```

File: main.go [hunk 3]
```diff
@@ -12,14 +13,15 @@ import (
...
-	RancherWorkloads   string `env:"RANCHER_WORKLOADS"`
+	Debug               bool   `env:"DEBUG"`
+	DisableOutput       bool   `env:"DISABLE_OUTPUT"`
+	TlsSkipVerification bool   `env:"TLS_SKIP_VERIFICATION"`
+	RancherBearerToken  string `env:"RANCHER_BEARER_TOKEN"`
+	RancherClusterId    string `env:"RANCHER_CLUSTER_ID"`
+	RancherNamespace    string `env:"RANCHER_NAMESPACE"`
+	RancherProjectId    string `env:"RANCHER_PROJECT_ID"`
+	RancherUrl          string `env:"RANCHER_URL"`
+	RancherWorkloads    string `env:"RANCHER_WORKLOADS"`
```
Comment (th0th) [inline main.go]: These are alphabetically sorted.
```suggestion
	Debug               bool   `env:"DEBUG"`
	DisableOutput       bool   `env:"DISABLE_OUTPUT"`
	RancherBearerToken  string `env:"RANCHER_BEARER_TOKEN"`
	RancherClusterId    string `env:"RANCHER_CLUSTER_ID"`
	RancherNamespace    string `env:"RANCHER_NAMESPACE"`
	RancherProjectId    string `env:"RANCHER_PROJECT_ID"`
	RancherUrl          string `env:"RANCHER_URL"`
	RancherWorkloads    string `env:"RANCHER_WORKLOADS"`
	TlsSkipVerification bool   `env:"TLS_SKIP_VERIFICATION"`
```

**Review by th0th — CHANGES_REQUESTED** [review]
I requested a few changes, then we can merge.

Also, did you have chance to run the version with changes? Did it work for your case?

**Comment by piechpatrick** [comment]
i hope all looks great atm

**Comment by th0th** [comment]
Thanks.

**Pull request merged**